core-build/node-library-build/src/index.ts

```diff
@@ -1,15 +1,20 @@
 // Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
 // See LICENSE in the project root for license information.
 
-import { task, watch, serial, parallel, IExecutable, setConfig, IBuildConfig } from '@microsoft/gulp-core-build';
+import { CopyTask, task, watch, serial, parallel, IExecutable, setConfig, IBuildConfig } from '@microsoft/gulp-core-build';
 import { typescript, tslint, apiExtractor } from '@microsoft/gulp-core-build-typescript';
 import { instrument, mocha } from '@microsoft/gulp-core-build-mocha';
 
 export * from '@microsoft/gulp-core-build';
 export * from '@microsoft/gulp-core-build-typescript';
 export * from '@microsoft/gulp-core-build-mocha';
 
-// Define default task groups.
+// pre copy and post copy allows you to specify a map of dest: [sources] to copy from one place to another.
+export const preCopy: CopyTask = new CopyTask();
+preCopy.name = 'pre-copy';
+
+export const postCopy: CopyTask = new CopyTask();
+postCopy.name = 'post-copy';
```

Comment:
Originally I needed it because **api-documenter** was being copied into **api-extractor**'s folder as a code sample.

However, Ian convinced me that this was a weird design, I removed it.  However, the postCopy task is a pretty useful feature, so we decided to keep it in this PR.

---
In reply to: [134342219](https://github.com/Microsoft/web-build-tools/pull/325#discussion_r134342219) [](ancestors = 134342219)